web/src/main/webapp/WEB-INF/config-security/config-security.properties

```diff
@@ -19,14 +19,14 @@ ldap.base.dn.pattern=uid={0},${ldap.base.search.base}
 # Define if groups and profile information are imported from LDAP. If not, local database is used.
 # When a new user connect first, the default profile is assigned. A user administrator can update
 # privilege information.
-ldap.privilege.import=false
+ldap.privilege.import=true
```

Comment:
I would prefer to keep the default to false as usually users don't have privileges info in LDAPs and use GN admin for that.